src/hooks/useViewportRows.ts

```diff
@@ -91,6 +93,17 @@ export function useViewportRows<R>({
   const isGroupRow = <R>(row: unknown): row is GroupRow<R> => allGroupRows.has(row);
 
   const overscanThreshold = 4;
+
+  if (!enableVirtualization) {
+    return {
+      rowOverscanStartIdx: 0,
+      rowOverscanEndIdx: rows.length - 1,
+      rows,
+      rowsCount,
+      isGroupRow
+    };
+  }
+
```

Comment:
Can keep `overscanThreshold` after the `if`

```suggestion
  if (!enableVirtualization) {
    return {
      rowOverscanStartIdx: 0,
      rowOverscanEndIdx: rows.length - 1,
      rows,
      rowsCount,
      isGroupRow
    };
  }

  const overscanThreshold = 4;
```